src/main/java/frc/utils/beambreaker/digitalbeambreaker/DigitalBeamBreaker.java

```diff
@@ -0,0 +1,24 @@
+package frc.utils.beambreaker.digitalbeambreaker;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.utils.beambreaker.BeamBreakerInputsAutoLogged;
+import frc.utils.beambreaker.IBeamBreaker;
+
+public class DigitalBeamBreaker implements IBeamBreaker {
+
+	private final DigitalInput beambreaker;
+
+	private final Debouncer debouncer;
+
+	public DigitalBeamBreaker(int DigitalInputChannel, double debounceTime, Debouncer.DebounceType DebounceType) {
```

Comment:
```suggestion
	public DigitalBeamBreaker(int channel, double debounceTime, Debouncer.DebounceType debounceType) {
```
params always lower letter